modules/mailer/mailer.go

```
@@ -29,24 +30,24 @@ type Message struct {
 }
 
 // NewMessageFrom creates new mail message object with custom From header.
-func NewMessageFrom(to []string, from, subject, htmlBody string) *Message {
-	log.Trace("NewMessageFrom (htmlBody):\n%s", htmlBody)
+func NewMessageFrom(to []string, from, subject, body string) *Message {
+	log.Trace("NewMessageFrom (body):\n%s", body)
 
 	msg := gomail.NewMessage()
 	msg.SetHeader("From", from)
 	msg.SetHeader("To", to...)
 	msg.SetHeader("Subject", subject)
 	msg.SetDateHeader("Date", time.Now())
 
-	body, err := html2text.FromString(htmlBody)
-	if err != nil {
-		log.Error(4, "html2text.FromString: %v", err)
-		msg.SetBody("text/html", htmlBody)
-	} else {
-		msg.SetBody("text/plain", body)
-		if setting.MailService.EnableHTMLAlternative {
-			msg.AddAlternative("text/html", htmlBody)
+	plainBody, _ := html2text.FromString(body)
```

Why do we ignore the error?

---

good catch :)

```go
if err != nil || setting.MailService.SendAsPlainText {
```
should do the trick :)

---

Not sure why I did that, but I have changed it now :)